inc/member-directory.php

```diff
@@ -0,0 +1,330 @@
+		'separate_items_with_commas' => __( 'Separate items with commas', 'inn' ),
+		'add_or_remove_items'        => __( 'Add or remove items', 'inn' ),
+		'choose_from_most_used'      => __( 'Choose from the most used', 'inn' ),
+		'popular_items'              => __( 'Popular Items', 'inn' ),
+		'search_items'               => __( 'Search Items', 'inn' ),
+		'not_found'                  => __( 'Not Found', 'inn' ),
+		'no_terms'                   => __( 'No items', 'inn' ),
+		'items_list'                 => __( 'Items list', 'inn' ),
+		'items_list_navigation'      => __( 'Items list navigation', 'inn' ),
+	);
+	$member_directory_args = array(
+		'labels'                     => $member_directory_labels,
+		'hierarchical'               => true,
+		'public'                     => true,
+		'show_ui'                    => true,
+		'show_admin_column'          => true,
+		'show_in_nav_menus'          => true,
+		'show_tagcloud'              => true,
+	);
+	register_taxonomy( 'ppu_focus_areas', array( 'inn_member' ), $member_directory_args );
+
+	$member_project_labels = array(
+		'name'                       => _x( 'Projects', 'inn' ),
+		'singular_name'              => _x( 'Project', 'inn' ),
+		'menu_name'                  => __( 'Projects', 'inn' ),
+		'all_items'                  => __( 'All Projects', 'inn' ),
+		'parent_item'                => __( 'Parent Item', 'inn' ),
+		'parent_item_colon'          => __( 'Parent Item:', 'inn' ),
+		'new_item_name'              => __( 'New Project', 'inn' ),
+		'add_new_item'               => __( 'Add New Project', 'inn' ),
+		'edit_item'                  => __( 'Edit Project', 'inn' ),
+		'update_item'                => __( 'Update Project', 'inn' ),
+		'view_item'                  => __( 'View Projects', 'inn' ),
+		'separate_items_with_commas' => __( 'Separate items with commas', 'inn' ),
+		'add_or_remove_items'        => __( 'Add or remove items', 'inn' ),
+		'choose_from_most_used'      => __( 'Choose from the most used', 'inn' ),
+		'popular_items'              => __( 'Popular Items', 'inn' ),
+		'search_items'               => __( 'Search Items', 'inn' ),
+		'not_found'                  => __( 'Not Found', 'inn' ),
+		'no_terms'                   => __( 'No items', 'inn' ),
+		'items_list'                 => __( 'Items list', 'inn' ),
+		'items_list_navigation'      => __( 'Items list navigation', 'inn' ),
+	);
+
+	$member_project_args = array(
+		'labels'                     => $member_project_labels,
+		'hierarchical'               => true,
+		'public'                     => true,
+		'show_ui'                    => true,
+		'show_admin_column'          => true,
+		'show_in_nav_menus'          => true,
+		'show_tagcloud'              => true,
+	);
+	register_taxonomy( 'pauinn_project_tax', array( 'pauinn_project', 'pp_opportunity', 'post', 'inn_member' ), $member_project_args );
+}
+add_action( 'init', 'member_directory_tax', 0 );
+
+add_action( 'cmb2_admin_init', 'inn_member_info' );
```

Comment:
You reference cmb2 but it's not in the theme or in plugins in the umbrella? 

I don't see the inn_member_info admin box on the "Edit member" page.